update/update_processes.cf

```diff
@@ -366,6 +372,22 @@ bundle agent maintain_cfe_windows
 
 }
 
+bundle agent maintain_cfe_systemd
+{
+  classes:
+    systemd::
+      "restart_cfe"
+      not => returnszero("/usr/bin/systemctl -q is-active cfengine3", "noshell"),
+      comment => "Check running status of CFEngine using systemd",
+      handle => "cfe_internal_maintain_cfe_systemd_classes_restart_cfe";
+
+  commands:
+    restart_cfe::
+      "/usr/bin/systemctl -q start cfengine3"
```

Comment:
I opted not to, because we enable it during package install, and the user might have removed CFEngine from the startup on purpose. Unlikely I guess, but it's how it works for the non-systemd platforms.